test/_files/cli_output.txt

```diff
@@ -0,0 +1 @@
+Howdy LeBron, CLI rox
```

Comment:
I'm thinking this file shouldn't be commited, it would even be better if it were deleted after the tests has run.

When this file exists in the repo, your changes in ./bin/mustache could actually be reverted, and the test below would still pass - right?